TarSCM/scm/git.py

```diff
@@ -251,7 +251,7 @@ def prepare_working_copy(self):
         # between multiple services
         org_clone_dir  = self.clone_dir
         self.clone_dir = self.repodir
-        command = ['git', 'clone', self.repocachedir, self.url, self.clone_dir]
+        command = ['git', 'clone', '--no-checkout']
```

Comment:
Thanks, I'll try that now.

As far as I can see this function is only called here:
https://github.com/openSUSE/obs-service-tar_scm/blob/master/TarSCM/scm/base.py#L80
And immediately afterwards a git reset --hard is done to the selected revision (or the program errors out if the revision is not found). So it looked to me like the checkout was redundant and would be immediately overwritte, and in theory it shouldn't affect that workflow either. But I'll check to make sure.